pi4j-core/src/main/java/com/pi4j/io/gpio/analog/AnalogOutputProvider.java

```diff
@@ -40,58 +40,58 @@ public interface AnalogOutputProvider extends AnalogProvider<AnalogOutputProvide
     /**
      * <p>create.</p>
      *
-     * @param address a {@link java.lang.Integer} object.
+     * @param pin a {@link java.lang.Integer} object.
      * @param <T> a T object.
      * @return a T object.
      */
-    default <T extends AnalogOutput> T create(Integer address) {
+    default <T extends AnalogOutput> T create(Integer pin) {
```

Comment:
If we are talking about pin, do we now mean the hardware pin number and not BCM address? E.g. Pin 15 here: https://pinout.xyz/pinout/pin15_gpio22/

Reply:
Not sure what the best solution is...

Current: `Integer pin();`

Possible alternatives:

1/ One setter method with a record
```
record Pin(NumberingScheme ns, Integer value)
enum NumberingScheme { BCM, PHYSICAL, ... } 
 
Pin pin();
```

2/ Private values set by two different methods (later more?)
```
private NumberingScheme schema
private value

Integer bcm(int value) {
   schema = NumberingScheme.BCM;
   value = value;
}
Integer physical(int value) {
   schema = NumberingScheme.PHYSICAL;
   value = value;
}
```

What do you think?